kombu/transport/SQS.py

```diff
@@ -160,15 +152,23 @@ def _reset_cycle(self):
 
     def entity_name(self, name, table=CHARS_REPLACE_TABLE):
         """Format AMQP queue name into a legal SQS queue name."""
-        return text_t(safe_str(name)).translate(table)
```

Comment:
I think FIFO support is absolutely a requirement. Otherwise Kombu will likely break if someone tries to use a new-style queue. However I did not implement this, I'm including the changes from @jseutter in my PR because they're based off of his branch.